pym/bob/input.py

```diff
@@ -59,7 +59,7 @@ def __maybeGlob(pred):
 
 def maybeGlob(pattern):
     if isinstance(pattern, list):
-        return [ __maybeGlob(p) for p in pattern ]
+        return [ __maybeGlob(p) for p in pattern ] if len(pattern) != 0 else None
```

Comment:
That should be handled before calling `maybeGlob`. Otherwise the `filter` keyword will behave differently.